### PR TITLE
Add lung disease terms

### PR DESCRIPTION
**Pull request by kallia-p**
fixes 3026

### CODE REVIEW

**Comment by kallia-p** [comment]
fixes #3026

**Comment by kallia-p** [comment]
@nicolevasilevsky Also failed!

**Comment by nicolevasilevsky** [comment]
no worries- I will take a look asap. Thanks for working on this!

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -435463,6 +435463,93 @@ synonym: "attenuated hypophosphatasia" EXACT [https://www.clinicalgenome.org/aff
...
+
+[Term]
+id: MONDO:0600019
+name: acinar dysplasia caused by mutation in TBX4
+def: "Any acinar dysplasia in which the cause of the disease is a mutation in the TBX4 gene." [https://orcid.org/0000-0003-0113-912X, MONDO:patterns/disease_series_by_gene]
+synonym: "TBX4 acinar dysplasia" EXACT [MONDO:patterns/disease_series_by_gene]
+synonym: "TBX4 related acinar dysplasia" EXACT [MONDO:patterns/disease_series_by_gene]
+is_a: MONDO:0600016 {source="https://orcid.org/0000-0003-0113-912X"} ! acinar dysplasia
+intersection_of: MONDO:0600016 ! acinar dysplasia
+intersection_of: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/3666 ! FGF10
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:435520]: this should be TBX4, not FGF10

**Review by nicolevasilevsky — CHANGES_REQUESTED** [review]

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -435463,6 +435463,93 @@ synonym: "attenuated hypophosphatasia" EXACT [https://www.clinicalgenome.org/aff
 is_a: MONDO:0018570 {source="PMID:32973344", source="https://www.clinicalgenome.org/affiliation/40065/"} ! hypophosphatasia
 property_value: created_by http://orcid.org/0000-0002-5460-8025
 
+[Term]
+id: MONDO:0600014
+name: alveolar capillary dysplasia without misalignment
+def: "Alveolar capillary dysplasia (ACD) without misalignement of the pulmonary veins is a rare form of interstitial and vascular lung disease that presents as severe pulmonary hypertension and refractory hypoxemia early in life." [https://orcid.org/0000-0003-0113-912X, https://www.sciencedirect.com/topics/medicine-and-dentistry/alveolar-capillary-dysplasia]
+synonym: "ACD without misalignment" EXACT ABBREVIATION []
+is_a: MONDO:0020295 {source="https://orcid.org/0000-0003-0113-912X"} ! congenital pulmonary veins anomaly
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:435471]: is there not a more immediate concept that unites these two? eg "alveolar capillary dysplasia"?

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -435463,6 +435463,93 @@ synonym: "attenuated hypophosphatasia" EXACT [https://www.clinicalgenome.org/aff
...
+name: alveolar capillary dysplasia with misalignment
+def: "Alveolar capillary dysplasia (ACD) with misalignment of the pulmonary veins is a rare form of interstitial and vascular lung disease that presents as severe pulmonary hypertension and refractory hypoxemia early in life." [https://orcid.org/0000-0003-0113-912X, https://www.sciencedirect.com/topics/medicine-and-dentistry/alveolar-capillary-dysplasia]
+synonym: "ACD with misalignment" EXACT ABBREVIATION []
+is_a: MONDO:0020295 {source="https://orcid.org/0000-0003-0113-912X"} ! congenital pulmonary veins anomaly
+property_value: created_by http://orcid.org/0000-0002-5460-8025
+
+[Term]
+id: MONDO:0600016
+name: acinar dysplasia
+def: "Congenital acinar dysplasia (AcDys) is a lethal, developmental lung malformation resulting in neonatal respiratory insufficiency. This entity is characterized by pulmonary hypoplasia and arrest in the pseudoglandular stage of development, resulting in the absence of functional gas exchange." [https://orcid.org/0000-0003-0113-912X, PMID:31641660]
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:435485]: Avoid repeating the term name in the definition 

and if the term name is repeated, make sure it it is consistent with the actual term name!

e..g is this congenital AcDys? is there a non-congenital AcDys?

https://douroucouli.wordpress.com/2019/07/08/ontotip-write-simple-concise-clear-operational-textual-definitions/

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -435463,6 +435463,93 @@ synonym: "attenuated hypophosphatasia" EXACT [https://www.clinicalgenome.org/aff
...
+synonym: "AcDys" EXACT [] {synonymtypedef="ABBREVIATION"}
+is_a: MONDO:0015930 {source="https://orcid.org/0000-0003-0113-912X"} ! respiratory malformation
+property_value: created_by http://orcid.org/0000-0002-5460-8025
+
+[Term]
+id: MONDO:0600017
+name: acinar dysplasia caused by mutation in FGF10
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:435492]: Do we know why these are not in OMIM? @nicolevasilevsky do we have an SOP for requesting them? Anything that conforms to this DP is a candidate for a PS member

**Review by cmungall — CHANGES_REQUESTED** [review]
This may actually be a complex PR

we have this concept:

```
id: MONDO:0009934
name: congenital alveolar capillary dysplasia
def: "Congenital alveolar capillary dysplasia (ACD) is a rare and fatal developmental lung disease characterized by respiratory distress in neonates due to refractory hypoxemia and severe pulmonary arterial hypertension." [Orphanet:210122]
comment: Editor note: MESH and NCIT split this, see NCIT:C98809
subset: gard_rare
subset: ordo_disease {source="Orphanet:210122"}
synonym: "ACDMPV" EXACT ABBREVIATION [MESH:C536590, MONDO:Lexical, OMIM:265380, Orphanet:210122]
synonym: "alveolar capillary dysplasia" EXACT [NCIT:C98809]
synonym: "alveolar capillary dysplasia with misalignment of pulmonary veins" EXACT [GARD:0008644, MESH:C536590, MONDO:Lexical, OMIM:265380, Orphanet:210122]
synonym: "alveolar capillary dysplasia with misalignment of pulmonary veins and Other congenital anomalies" RELATED [OMIM:265380]
synonym: "alveolar capillary dysplasia with misalignment of pulmonary veins and other congenital anomalies" RELATED [MESH:C536590]
synonym: "alveolar capillary dysplasia with misalignment of pulmonary vessels" EXACT [Orphanet:210122]
synonym: "alveolar capillary dysplasia with pulmonary venous misalignment" RELATED [GARD:0008644]
synonym: "congenital alveolar capillary dysplasia" EXACT [GARD:0008644, MESH:C536590]
synonym: "congenital alveolar capillary dysplasia with misalignment of pulmonary veins" EXACT [DOID:13042]
synonym: "familial persistent pulmonary hypertension of the newborn" RELATED [GARD:0008644]
synonym: "fetal circulation" EXACT [DOID:13042]
synonym: "persistent fetal circulation" RELATED [DOID:13042, ICD9CM_2006:747.83]
synonym: "persistent fetal circulation syndrome" EXACT EXCLUDE [DOID:13042]
synonym: "persistent pulmonary hypertension of the newborn" RELATED [CSP2005:0723-5955, CSP2005:2595-6954, DOID:13042]
synonym: "pulmonary hypertension, familial persistent of the newborn" RELATED [GARD:0008644]
xref: COHD:197970 {source="MONDO:equivalentTo"}
xref: DOID:13042 {source="EFO:1001103", source="MONDO:equivalentTo"}
xref: EFO:1001103 {source="MONDO:equivalentTo"}
xref: GARD:0008644 {source="MONDO:equivalentTo"}
xref: ICD10:P29.3 {source="DOID:13042", source="EFO:1001103", source="MONDO:equivalentTo"}
xref: ICD9:747.49 {source="MONDO:relatedTo", source="i2s"}
xref: ICD9:747.83 {source="DOID:13042"}
xref: MedDRA:10054726 {source="EFO:1001103"}
xref: MESH:C536590 {source="MONDO:equivalentTo"}
xref: MESH:D010547 {source="DOID:13042", source="EFO:1001103", source="MONDO:equivalentTo"}
xref: NCIT:C85006 {source="MONDO:relatedTo", source="DOID:13042", source="EFO:1001103"}
xref: NCIT:C98809 {source="MONDO:equivalentTo"}
xref: OMIM:265380 {source="ORDO:210122/e", source="Orphanet:210122", source="DOID:13042", source="MONDO:equivalentTo", source="GARD:0008644"}
xref: Orphanet:210122 {source="OMIM:265380", source="MONDO:equivalentTo", source="GARD:0008644"}
xref: SCTID:204507004 {source="DOID:13042"}
xref: SCTID:206597007 {source="DOID:13042"}
xref: SCTID:233815004 {source="MONDO:relatedTo", source="DOID:13042"}
xref: SCTID:35604006 {source="DOID:13042", source="EFO:1001103"}
xref: SCTID:447275002 {source="MONDO:equivalentTo", source="GARD:0008644"}
xref: UMLS:C0031190 {source="OMIM:265380", source="NCBI:mim2gene_medline", source="DOID:13042", source="MONDO:equivalentTo"}
xref: UMLS:C2677362 {source="Orphanet:210122", source="MONDO:notFoundInDiseaseSubset", source="NCIT:C98809"}
xref: UMLS:C2960310 {source="MONDO:notFoundInDiseaseSubset", source="GARD:0008644"}
is_a: MONDO:0017017 {source="Orphanet:210122"} ! primary interstitial lung disease specific to childhood due to alveolar vascular disorder
is_a: MONDO:0020295 {source="MONDO:cjm", source="linkedlifedata"} ! congenital pulmonary veins anomaly
relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/3809 {source="mim2gene_medgen"} ! FOXF1
property_value: seeAlso https://rarediseases.info.nih.gov/diseases/8644/alveolar-capillary-dysplasia xsd:anyURI {source="GARD:0008644"}
```

but I think it's conflated. The equivalent OMIM is for a specific gene (FOX1 - https://omim.org/entry/265380). I think we want to obsolete-split this mondo and weave it into the work here. This is a good test of our documentation/SOPs! @nicolevasilevsky you may need to help a bit here

**Comment by nicolevasilevsky** [comment]
@cmungall I'll work on this with @kallia-p :)

**Comment by nicolevasilevsky** [comment]
I separated this into separate tickets and PRs, closing this.

**Pull request closed without merging**